test/regression.uts

```diff
@@ -1217,7 +1217,7 @@ in6_addrtomac("FE80::" + in6_mactoifaceid("FF:00:00:00:00:00", ulbit=0)) == 'ff:
 import socket
 
 res=True
-for _ in xrange(10):
+for a in xrange(10):
```

Comment:
That's a really hidden bug.
Using this replaces `_` by the `xrange` iterator, so once the for is finished, `_` will be stuck with the value `9` for ever.